use-cases/virgo/slurm_ray.sh

```diff
@@ -47,9 +48,13 @@ nodes_array=($nodes)
 head_node=${nodes_array[0]}
 port=7639       # This port will be used by Ray to communicate with worker nodes.
 
-# This is so that the ray.init() command called from the hpo.py script knows which ports to connect to
-export ip_head="$head_node"i:"$port"
-export head_node_ip="$head_node"i
+# This is so that the ray.init() command called from the hpo.py script knows
+# which ports to connect to
+export HEAD_NODE_PORT="$head_node"i:"$port"
```

Comment:
wait, here `HEAD_NODE_PORT` seems a concat of both IP and PORT.

I like the idea of having a varible called `HEAD_NODE_PORT` (see also two lines below for another example), but it must be just the port. In the python script you can concat HEAD_NODE_PORT and HEAD_NODE_IP to correctly pass the info to Ray

Reply:
Resolving because of below comment